ivy/functional/frontends/numpy/indexing_routines/indexing_like_operations.py

```diff
@@ -0,0 +1,10 @@
+import ivy
+
+
```

Comment:
You need to add `@to_ivy_arrays_and_back` here to stop getting the `__wrapped__` error.

Reply:
I called in the main function file by seeing ivy docs (alias function), is that right? but don't know about diagonal similar function to call as did in the docs.
` from ivy.functional.frontends.numpy.func_wrapper import to_ivy_arrays_and_back` 
error
`ivy.exceptions.IvyBackendException: numpy: diagonal: diagonal() takes 1 positional argument but 4 were given`

From the above error, it seems to be there is another function similar to diagonal.

Alias Function
https://lets-unify.ai/ivy/deep_dive/ivy_frontends_tests.html